llmfoundry/utils/exceptions.py

```diff
@@ -41,12 +41,28 @@
 EvalDataLoaderLocation = 'EvalDataloader'
 
 
-class ContextualError(Exception):
+class SerializableError():
+
+    def __getstate__(self):
+        return self.__dict__
+
+    def __setstate__(self, state: Dict[str, Any]):
+        for key, value in state.items():
+            setattr(self, key, value)
+
+
+class ContextualError(Exception, SerializableError):
     """Error thrown when an error occurs in the context of a specific task."""
 
     location: Optional[ErrorLocation] = None
     error_attribution: Optional[ErrorAttribution] = None
 
+    def __init__(self, message: str) -> None:
```

Comment:
Are these still needed? I'm thinking no?

Reply:
I tried removing it and it brought back the error :/

Reply:
oh sorry, you still need the constructor, but you don't need the __str__ override